django_u2f/models.py

```diff
@@ -15,7 +15,8 @@
 
 
 class U2FKey(models.Model):
-    user = models.ForeignKey(settings.AUTH_USER_MODEL, related_name='u2f_keys')
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, related_name='u2f_keys',
+                             on_delete=models.PROTECT)
```

Comment:
Why PROTECT? If the user is deleted we don't need their u2f keys anymore.

Reply:
Sorry, my bad.